src/NHSD.BuyingCatalogue.Ordering.Api/Extensions/OrderExtensions.cs

```diff
@@ -36,7 +36,7 @@ public static bool IsAdditionalServicesSectionComplete(this Order order)
 
         public static bool IsFundingSourceComplete(this Order order)
         {
-            return order?.FundingSourceOnlyGms is not null;
+            return order?.FundingSourceOnlyGms is not null && (order.HasSolution() || order.HasAdditionalService());
```

Comment:
An order can only have an additional service if it has a solution. However, an order can contain only an associated service (and no solutions) so I think we need `HasAssociatedService` instead of `HasAdditionalService`.

Reply:
Doh! apologies, confused Additional & Associated services - corrected.